roles/barbican/templates/etc/apache2/sites-available/barbican.conf

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
I think we should not use Apache as there is currently no apparent reason to. Keystone had to use Apache because federation depends on Apache modules. I think we should be fine if we have HAProxy talk directly to the uwsgi processes.